.ci/linux-steps.yaml

```diff
@@ -0,0 +1,18 @@
+steps:
+# Checkout repository
+- checkout: self
+  clean: true
+  fetchDepth: 1
+
+# Install protobuf
+- script: |
+    sudo apt-get install protobuf
```

Comment:
```suggestion
    sudo apt-get update
    sudo apt-get install -y protobuf
```

Reply:
threw same error not apt package found snap package found